packages/breadcrumbs/src/styled/StyledBreadcrumbItem.ts

```diff
@@ -6,23 +6,36 @@
  */
 
 import styled from 'styled-components';
-import classNames from 'classnames';
-import { retrieveComponentStyles } from '@zendeskgarden/react-theming';
-import BreadcrumbStyles from '@zendeskgarden/css-breadcrumbs';
+import { getColor } from '@zendeskgarden/react-theming';
 
 const COMPONENT_ID = 'breadcrumbs.item';
 
-export interface IStyledBreadcrumbItemProps {
-  isCurrent?: boolean;
-}
-
-export const StyledBreadcrumbItem = styled.li.attrs<IStyledBreadcrumbItemProps>(props => ({
+export const StyledBreadcrumbItem = styled.li.attrs(() => ({
   'data-garden-id': COMPONENT_ID,
-  'data-garden-version': PACKAGE_VERSION,
-  className: classNames(BreadcrumbStyles['c-breadcrumb__item'], {
-    // State
-    [BreadcrumbStyles['is-current']]: props.isCurrent
-  })
+  'data-garden-version': PACKAGE_VERSION
 }))<IStyledBreadcrumbItemProps>`
-  ${props => retrieveComponentStyles(COMPONENT_ID, props)};
+  ${props => `
+    line-height: ${props.theme.lineHeights.md};
+    color: ${props.isCurrent ? getColor(props.theme.colors.neutralHue, 600) : 'inherit'}
+    ${
+      props.isCurrent === undefined || props.isCurrent === false
+        ? `::after {
```

Comment:
Previously the `::before` pseudo element was used, but now `::after` is used.

I had a difficult time wrapping my head around using `::before` to position the chevrons. I opted for `::after` because all breadcrumbs need a chevron _after_ itself besides the current breadcrumb (or last breadcrumb).

Reply:
Use `::before` since `::after` has the affect of polluting the user space. See [internal-only link](https://github.com/zendeskgarden/css-forms/pull/22) for details. We tend to use `::after` only when needed:
- swapping `::before`/`::after` `background-image` on hover
- z-index styling in [css-arrows](https://github.com/zendeskgarden/css-components/tree/master/packages/arrows)

Reply:
Having said ☝️, it's going to make a lot more sense for you to move the chevron out into it's own DOM element. Compare tag [CSS](https://github.com/zendeskgarden/css-components/blob/master/packages/tags/src/_remove.css) with [CSS-in-JS](https://github.com/zendeskgarden/react-components/blob/next/packages/tags/src/elements/Close.tsx) for reference.

Unless it's going to be [tricky](https://github.com/zendeskgarden/react-components/blob/next/packages/forms/src/styled/radio/StyledRadioInput.js) we need to take this opportunity to inline SVGs. Unlike CSS, we control the DOM here and this instance should be fairly straightforward.

Reply:
Actually, this pattern might work better for you: https://github.com/zendeskgarden/react-components/blob/next/packages/buttons/src/styled/StyledExternalIcon.ts

Reply:
Regarding using `<svg>` for the separator:

> To prevent screen reader announcement of the visual separators between links, they are added via CSS:

> The separators are part of the visual presentation that signifies the breadcrumb trail, which is already semantically represented by the nav element with its label of Breadcrumb. So, using a display technique that is not represented in the accessibility tree used by screen readers prevents redundant and potentially distracting verbosity.

> Each link has a border on one side that is skewed with the CSS’ transform property so it resembles a slash.

Source: https://www.w3.org/TR/wai-aria-practices/examples/breadcrumb/index.html

An `aria-hidden` can be applied to the separator to prevent it from being picked up on screen readers for the `<svg>` approach. Another challenge with the `<svg>` approach is centering the chevron — managed to center it with `position:relative;top:1px;` which is not great but seems to work.